ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -166,10 +166,14 @@ public enum CocoaPodUtils {
     let podsDir = projectDir.appendingPathComponent("Pods")
     var installedPods: [String: PodInfo] = [:]
     for (podName, version) in pods {
-      let podDir = podsDir.appendingPathComponent(podName)
-      guard FileManager.default.directoryExists(at: podDir) else {
-        fatalError("Directory for \(podName) doesn't exist at \(podDir) - failed while getting " +
-          "information for installed Pods.")
+      var podDir: URL
+      podDir = podsDir.appendingPathComponent(podName)
+      if !FileManager.default.directoryExists(at: podDir) {
+        guard let repoDir = LaunchArgs.shared.localPodspecPath else {
```

Comment:
Nit: a comment here may be good to say something along the lines of, "The pod doesn't exist in a remote directory, check the local podspec path to see if a local podspec was provided."

I was confused when I first read the code assuming it'd always be required but after typing out a comment about it I realized the situation.